examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,1090 @@
+        "import nest_asyncio\n",
+        "\n",
+        "nest_asyncio.apply()\n",
+        "\n",
+        "from llama_index.evaluation import generate_question_context_pairs\n",
+        "from llama_index import VectorStoreIndex, SimpleDirectoryReader, ServiceContext\n",
+        "from llama_index.node_parser import SimpleNodeParser\n",
+        "from llama_index.evaluation import generate_question_context_pairs\n",
+        "from llama_index.evaluation import RetrieverEvaluator\n",
+        "from llama_index.llms import OpenAI\n",
+        "\n",
+        "import openai\n",
+        "import os\n",
+        "import pandas as pd"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GocbeIV3rQYc"
+      },
+      "source": [
+        "Set Your OpenAI API Key"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "bocDlS3FrP8L"
+      },
+      "outputs": [],
+      "source": [
+        "os.environ['OPENAI_API_KEY'] = 'YOUR OPENAI API KEY'"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oKXIZyFOKIIT"
+      },
+      "source": [
+        "Let's use Paul Graham Essay text for building RAG pipeline."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OjyT5I-kKPbl"
+      },
+      "source": [
+        "#### Download Data"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "UUOKqSSeCkEN",
+        "outputId": "17c6b9f6-f6f6-4d9f-d75f-a197133f15f1"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "--2023-11-11 13:59:14--  https://raw.githubusercontent.com/run-llama/llama_index/main/docs/examples/data/paul_graham/paul_graham_essay.txt\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.111.133, 185.199.110.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 75042 (73K) [text/plain]\n",
+            "Saving to: ‘data/paul_graham/paul_graham_essay.txt’\n",
+            "\n",
+            "data/paul_graham/pa 100%[===================>]  73.28K  --.-KB/s    in 0.03s   \n",
+            "\n",
+            "2023-11-11 13:59:14 (2.86 MB/s) - ‘data/paul_graham/paul_graham_essay.txt’ saved [75042/75042]\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "!mkdir -p 'data/paul_graham/'\n",
+        "!wget 'https://raw.githubusercontent.com/run-llama/llama_index/main/docs/examples/data/paul_graham/paul_graham_essay.txt' -O 'data/paul_graham/paul_graham_essay.txt'"
```

Comment:
Use curl instead - I its more widely supported by default (e.g. macs)

Reply:
nit 2: this output box isn't super critical to show, but I guess it could be good to sanity check the command worked. up to you to keep or remove :)